homeassistant/components/viaggiatreno/sensor.py

```diff
@@ -23,7 +23,7 @@
 ATTRIBUTION = "Powered by ViaggiaTreno Data"
 
 VIAGGIATRENO_ENDPOINT = (
-    "http://www.viaggiatreno.it/viaggiatrenonew/"
+	"http://www.viaggiatreno.it/infomobilita/"
```

Comment:
This integration needs refactoring; and the API/connection logic needs to be extracted into its own library (which is published on PyPi).